settlement/paypal/transform.go

```diff
@@ -0,0 +1,346 @@
+package paypal
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/brave-intl/bat-go/settlement"
+	"github.com/brave-intl/bat-go/utils/altcurrency"
+	uuid "github.com/satori/go.uuid"
+	"github.com/shopspring/decimal"
+)
+
+// CompleteSettlement marks the settlement file as complete
+func CompleteSettlement(args Args) error {
+	fmt.Println("RUNNING: complete")
+	if args.In == "" {
+		return errors.New("the 'in' flag must be set")
+	}
+	if args.Out == "./paypal-settlement" {
+		args.Out = "./paypal-settlement.json"
+		// return errors.New("the 'out' flag must be set")
+	}
+	payouts, err := ReadFiles(args.Out)
+	if err != nil {
+		return err
+	}
+	for i, payout := range *payouts {
+		payout.Status = "complete"
+		(*payouts)[i] = payout
+	}
+	fmt.Println(payouts)
+	err = WriteEyeshadeTransactions(args.Out, payouts)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+// CreateSettlementFile starts the transform process
+func CreateSettlementFile(args Args) (err error) {
+	fmt.Println("RUNNING: transform")
+	if args.In == "" {
+		return errors.New("the 'in' flag must be set")
+	}
+	if args.Currency == "" {
+		return errors.New("the 'currency' flag must be set")
+	}
+	// check that date is fine
+	_, err = args.CheckDate()
+	if err != nil {
+		return err
+	}
+
+	payouts, err := ReadFiles(args.In)
+	if err != nil {
+		return err
+	}
+
+	// TEMPORARY HACK. NEED UPDATED DATA STRUCTURE
+	for i, payout := range *payouts {
+		payout.Provider = "paypal"
+		(*payouts)[i] = payout
+	}
+
+	rate, err := GetRate(args)
+	if err != nil {
+		return err
+	}
+	args.Rate = rate
+
+	txs, err := CreateEyeshadeTransactions(args, payouts)
```

Comment:
this doesn't actually create eyeshade transactions?

Reply:
it creates the json. eyeshade tx was a struct that was used in previous iterations